cypress/support/step_definition/tasks/createEditAndDeleteTasks.spec.js

```diff
@@ -102,21 +58,6 @@ And('I click on Submit Public Submission in confirmation window on task page', (
   tasks.clickSubmitInConfirmationWindow()
 })
 
-// And ('I click on Draft Checkbox', () => {
```

Comment:
And ('I click on Draft Checkbox', () => {} -> this step can be defined inside the 'commonTasksSteps.spec.js' because its used in two scenarios in this feature.